From a8bbf741d15a1c887b8ad80a72fb85c4b1c3d61d Mon Sep 17 00:00:00 2001
From: Robin Gong <b38343@freescale.com>
Date: Wed, 2 Apr 2014 17:00:17 +0800
Subject: [PATCH] ENGR00306653-3: ARM: imx_v7_defconfig enable snvs_pwrkey
 driver by default

enable snvs_pwrkey driver by default

Signed-off-by: Robin Gong <b38343@freescale.com>
(cherry picked from commit 07952e49e72e62d8d6fcea437caf46f4ca0ebb86)
---
 arch/arm/configs/imx_v7_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/configs/imx_v7_defconfig b/arch/arm/configs/imx_v7_defconfig
index e4e63a9..70ff2e5 100644
--- a/arch/arm/configs/imx_v7_defconfig
+++ b/arch/arm/configs/imx_v7_defconfig
@@ -133,6 +133,7 @@ CONFIG_BRCMFMAC=m
 CONFIG_INPUT_EVDEV=y
 CONFIG_INPUT_EVBUG=m
 CONFIG_KEYBOARD_GPIO=y
+CONFIG_KEYBOARD_SNVS_PWRKEY=y
 CONFIG_KEYBOARD_IMX=y
 CONFIG_MOUSE_PS2=m
 CONFIG_MOUSE_PS2_ELANTECH=y
-- 
1.8.0

